dlab_core/app.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
```

Comment:
better to use api/app.py location for this file because api is not a part of dlab_core library

Reply:
moved